features-support/hooks/before.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+/* eslint new-cap: 0 */
+
+module.exports = function beforeHooks() {
+
+  // Remove any old test data.
+  this.Before(function(callback) {
```

Comment:
Would this not run before each test? I would only do it once before running a test suite